src/ILAMB/Confrontation.py

```diff
@@ -996,8 +996,9 @@ def _scoreFunction(ref,com):
         def _plotDistribution(dist,xedges,yedges,xlabel,ylabel,filename):
             fig,ax = plt.subplots(figsize=(6,5.25),tight_layout=True)
             pc = ax.pcolormesh(xedges, yedges, dist,
-                               norm = LogNorm(vmin = 1e-4,vmax = 1e-1),
-                               cmap = 'plasma' if 'plasma' in plt.cm.cmap_d else 'summer')
+                               norm = LogNorm(),
+                               cmap = 'plasma' if 'plasma' in plt.cm.cmap_d else 'summer',
+                               vmin = 1e-4, vmax = 1e-1)
```

Comment:
Was there some problem with the way I had this? Sometimes things like this are this way to avoid warnings in the output.

Reply:
This is also just a carryover from ILAMB 2.5.